src/pretix/plugins/banktransfer/payment.py

```diff
@@ -116,16 +121,7 @@ def test_mode_message(self):
 
     @property
     def settings_form_fields(self):
-        d = OrderedDict(
-            list(super().settings_form_fields.items()) + list(BankTransfer.form_fields().items()) + [
-                ('omit_hyphen', forms.BooleanField(
-                    label=_('Do not include a hypen in the payment reference.'),
-                    help_text=_('This is required in some countries.'),
-                    required=False
-                )),
-
-            ]
-        )
+        d = OrderedDict(list(super().settings_form_fields.items()) + list(BankTransfer.form_fields().items()))
```

Comment:
This is also on purpose. This hyphen thing is a very specific option, only really a problem in Finland. With your change, we would start asking it to every user who goes through the quick event setup.

Reply:
I searched for usages of `BankTransfer.form_fields()`. In https://github.com/pretix/pretix/blob/76e75bef65a9543594ac418e0ebc16f61a7f9a0e/src/pretix/control/forms/event.py#L1495

the fields shown in the quick setup are hardcoded, so I thought that wouldn't be a problem.

Reply:
Agreed